Iniciante/1116/README.md

```diff
@@ -0,0 +1,45 @@
+# Problema:
+
+Escreva um algoritmo que leia 2 números e imprima o resultado da divisão do primeiro pelo segundo. Caso não for possível mostre a mensagem “divisao impossivel” para os valores em questão.
+
+##### Problema completo: https://www.urionlinejudge.com.br/judge/pt/problems/view/1116
+
+# Resolução:
+
+O objetivo desse exercício é ler dois valores N vezes, efetuar a divisão caso seja possível e mostrar o resultado na tela.
+
+Para começar criamos quatro variáveis do tipo `int`:
+```c
+    int x, y, N, i;
+```
+`x` e `y` representam os valores do dividendo e divisor, respectivamente. `N` representa o número de cálculos que serão feitos, `i` é uma variável auxiliar que será usada no `for`.
+
+Temos que saber quantos casos serão feitos, para isso lemos `N` usando `scanf`:
+```c
+    scanf("%d", &N);
+```
+Com a variável lida, fazemos um laço de repetição usando `for`. Igualamos `i` a 0 (`i=0`) e incrementamos o valor dessa variável (`i++`) a cada repetição até que `i` seja igual a N-1 (`i<N`), que é o menor valor inteiro antes de `N`:
```

Comment:
acho q isso ta explicando ta indo muito deep na sintaxe do for, não vejo necessidade